test/unit/routingRoundTrip.js

```diff
@@ -0,0 +1,126 @@
+import _ from 'lodash';
```

Comment:
Should this file actually be in the `test/unit` subfolder?  It doesn't seem like a unit test to me.